src/provider/github/client.ts

```diff
@@ -14,8 +14,9 @@ export class GithubClient implements Client {
 
   public name = 'GitHub Client';
 
-  constructor(protocol: string, hostname: string, token: string, logger: (message: string) => void) {
-    this.client = getClient(this.getApiEndpoint(protocol, hostname), token, logger);
+  constructor(protocol: string, hostname: string, token: string, logger: (message: string) => void,
+              rejectUnauthoized = false) {
```

Comment:
This should default to `true` right now I think.
Default default should be to reject invalid ssl chains.